.gitlab/llnl-ci.yml

```diff
@@ -1,173 +1,9 @@
+include:
+  - local: 'llnl-ci-pip.yml'
+  - local: 'llnl-ci-conda.yml'
+  - local: 'llnl-ci-spack.yml'
```

Comment:
`include: '.gitlab/llnl-ci-*.yml'`?

Reply:
Done

Reply:
still missing parts, should be `include: '.gitlab/llnl-ci-*.yml'`

Reply:
Done